server/instance.go

```diff
@@ -357,6 +357,11 @@ func (s *Server) registerInstanceRoutes(g *echo.Group) {
 		}
 		entry := list[0]
 
+		lastRecordedSchema, err := db.FindLastRecordedSchema(ctx, driver, entry.Version)
+		if err != nil {
+			return echo.NewHTTPError(http.StatusInternalServerError, "Failed to fetch migration history list").SetInternal(err)
```

Comment:
Could be more descriptive.

Failed to fetch last recorded schema before version: %v

same below